content/kotlin/concepts/strings/strings.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Strings'
+Description: 'A string in Kotlin is an object that holds a sequence of characters contained within a pair of double quotes (").'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+In Kotlin, a `string` is an immutable object that represents a sequence of characters.
+
+## Syntax
+
+```pseudo
+val stringName: String = "string value"
+```
+
+## Example
+
+You can also use `string` templates to include dynamic values in your strings. To do this, you enclose the dynamic value in curly braces and precede it with a dollar sign, like this:
```

Comment:
You should separate string templates into its own section after the example. Call it "## String Templates" and provide examples that produce some sort of output.

Reply:
@SSwiniarski hmmm alright so you want me to keep this string template section separate from the example section? like I can remove all this and make a diff header as string templates like you said

Reply:
alright check now